### PR TITLE
[cli] Remove EAS Build related code

### PR DESCRIPTION
**Pull request by brentvatne**
This all lives in eas-cli now!

### CODE REVIEW

**Review by EvanBacon — APPROVED** [review]
👍

File: packages/expo-cli/scripts/introspect.ts [hunk 1]
```diff
@@ -125,7 +125,7 @@ function formatCommandsAsMarkdown(commands: CommandData[]) {
   return Object.entries(grouped)
     .map(([groupName, commands]) => {
       // Omit internal and eas commands from the docs
```
Comment (dsokal) [inline packages/expo-cli/scripts/introspect.ts:127]:
```suggestion
      // Omit internal commands from the docs
```

**Review by dsokal — APPROVED** [review]
👏

**Review by wkozyra95 — APPROVED** [review]

**Comment by brentvatne** [comment]
we should also update the ["Only single target iOS apps are supported"](https://docs.expo.io/build-reference/limitations/) section in limitations to clarify that it only applies to automatically managed credentials and that you can do this through local credentials now

**Pull request merged**